File: Python/SivitasAkademik.py
# Import file:
from Human import Human

# Kelas SivitasAkademik (Child) - Mahasiswa (Parent)
class SivitasAkademik(Human):
    __asal_univ = ""
    __email_edu = ""

    # Konstuktor
    def __init__(self, asal_univ, email_edu):
        self.asal_univ = asal_univ
        self.email_edu = email_edu

    # Setter
    def set_asal_univ(self, asal_univ):
        self.asal_univ = asal_univ

    def set_email_edu(self, email_edu):
        self.email_edu = email_edu

    # Getter
    def get_asal_univ(self):
        return self.asal_univ
    
    def get_email_edu(self):
        return self.email_edu

File: PHP/SivitasAkademik.php
<?php 

require("Human.php");

class SivitasAkademik extends Human
{
    // Private attribute:
    private $asal_univ = '';
    private $email_edu = '';

    // Konstruktor
    public function __construct($nik = '', $nama = '', $jenis_kelamin = '', $asal_univ = '', $email_edu = '')
    {
        parent::__construct($nik, $nama, $jenis_kelamin);
        $this->asal_univ = $asal_univ;
        $this->email_edu = $email_edu;
    }

    // Setter
    // Asal Universitas
    public function set_asal_univ($asal_univ)
    {
        $this->asal_univ = $asal_univ;
    }
    // Email EDU
    public function set_email_edu($email_edu)
    {
        $this->email_edu = $email_edu;
    }

    // Getter
    // Asal Universitas
    public function get_asal_univ()
    {
        return $this->asal_univ;
    }
    // Email EDU
    public function get_email_edu()
    {
        return $this->email_edu;
    }

    // Destruktor
    public function __destruct()
    {
        
    }
}

?>